.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,55 @@
+# docs ref: https://github.com/helm/chart-testing-action
+name: Lint and Test Charts
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --chart-dirs charts)
```

Comment:
what is the comand `ct` ?

Reply:
`ct` is equal to `chart testing`, a command-line tool which is maintained by `helm`.

[Here](https://github.com/helm/chart-testing) is the repository of `ct`.